github/connection_config.go

```diff
@@ -5,8 +5,11 @@ import (
 )
 
 type githubConfig struct {
-	Token   *string `hcl:"token"`
-	BaseURL *string `hcl:"base_url"`
+	Token          *string `hcl:"token,optional"`
+	BaseURL        *string `hcl:"base_url,optional"`
+	AppId          *int64  `hcl:"app_id,optional"`
```

Comment:
Since these are IDs, we should make them strings (we don't know how long they'll be in the future)